microsoft-edge/webview2/index.md

```diff
@@ -84,16 +84,24 @@ WebView2 apps can run on the following versions of Windows:
 *  Windows 10 IoT Enterprise LTSC x32 2019
 *  Windows 10 IoT Enterprise LTSC x64 2019
 *  Windows 10 IoT Enterprise 21h1 x64
-*  Windows 8.1
-*  Windows 7 \*\*
 *  Windows Server 2022
 *  Windows Server 2019
 *  Windows Server 2016
-*  Windows Server 2012
+
+
+#### Windows 7 and 8
+
+WebView2 Runtime version 109, and WebView2 SDK version 1.0.####.0, is the final version that supports the following versions of Windows.  WebView2 Runtime version 110.0.1519.0 and higher, and WebView2 SDK version 1.0.1519.0 and higher, doesn't support these operating systems.
```

Comment:
@champnic @nishitha-burman - At `####`, what's the final SDK version # that supports Win7/8?

Reply:
Removed unknown version #